tensorbay/cli/commit.py

```diff
@@ -32,13 +34,10 @@ def _implement_commit(obj: Dict[str, str], tbrn: str, message: Tuple[str, ...])
         error(f'To commit, "{info}" must be in draft status, like "{info}#1"')
 
     dataset_client.checkout(draft_number=info.draft_number)
-    if message:
-        title, description = message[0], "\n".join(message[1:])
-    else:
-        title, description = edit_input(_COMMIT_HINT.format(dataset_client.get_draft().title))
 
-    if not title:
-        error("Aborting commit due to empty commit message")
+    draft = dataset_client.get_draft()
+    hint_message = format_hint(draft.title, draft.description, _COMMIT_HINT)
+    title, description = edit_draft_info(message, _COMMIT_ERROR, hint_message)
```

Comment:
`edit_draft_info` used in `commit` function?
should `edit_draft_info` need a better name?